diff --git a/node_modules/@types/passport/index.d.ts b/node_modules/@types/passport/index.d.ts
index 6154267..9f56fc3 100644
--- a/node_modules/@types/passport/index.d.ts
+++ b/node_modules/@types/passport/index.d.ts
@@ -5,7 +5,7 @@ declare global {
         // eslint-disable-next-line @typescript-eslint/no-empty-interface
         interface AuthInfo {}
         // eslint-disable-next-line @typescript-eslint/no-empty-interface
-        interface User {}
+        interface User {userId}
 
         interface Request {
             authInfo?: AuthInfo | undefined;
